src/arguments.js

```diff
@@ -4,17 +4,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { program, Option } from 'commander';
-import { exit } from 'process';
-import ora from 'ora';
-import fs from 'fs';
-import { AUTH, CLI_COMMAND_NAME, DEFAULT_AUTH, DEFAULT_FILENAME, DEFAULT_FORMAT, DEFAULT_MIN_HEIGHT, DEFAULT_TENANT, DEFAULT_WIDTH, ENV_VAR, FORMAT, TRANSPORT_TYPE, DEFAULT_EMAIL_SUBJECT, DEFAULT_EMAIL_NOTE } from './constants.js';
-import dotenv from "dotenv";
+var { program, Option } = require('commander');
+var { exit } = require('process');
```

Comment:
could you change `var` to `const`?

Reply:
Thanks! changed var to const in all files.